shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/java/org/apache/shardingsphere/sql/parser/integrate/jaxb/domain/statement/dal/SetVariableStatementTestCase.java

```diff
@@ -30,6 +30,9 @@
 @Setter
 public final class SetVariableStatementTestCase extends SQLParserTestCase {
     
-    @XmlAttribute(name = "variable")
-    private String variable;
+    @XmlAttribute(name = "variableName")
+    private String variableName;
+    
+    @XmlAttribute(name = "variableValue")
```

Comment:
Please reference other name of xml attribute, we prefer using `-` to delimiter tow words.

Reply:
done